src/frontend/src/app/home/home.component.html

```diff
@@ -25,15 +25,6 @@
           <p class="control">
               <input value="{{ entry.value }}" class="input" type="text" disabled>
           </p>
-          <p class="control">
```

Comment:
kindly fix home.component.ts too

Reply:
Sure I can but scope of this PR was to remove gauge only.

Reply:
I think
       ` this.options['data-type'] = 'radial-gauge';`
is in scope!

Reply:
not sure if any other?

Reply:
I have removed that missed line. I miss understood your comment `kindly fix home.component.ts` bit. Fixed now and I hope it will be production ready now.

Reply:
NP :)